app/Models/MessageSetting.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Gladiator\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class MessageSetting extends Model
+{
+    /**
+     * Get the fillable attributes for the model.
+     *
+     * @return array
+     */
+    protected $fillable = ['type', 'key', 'label', 'subject', 'body', 'pro_tip', 'signoff', 'shoutout'];
```

Comment:
@deadlybutter just a note here, you'll want to pull shoutout out of here when you make the pr for #280